pkg/docker-cli/Dockerfile

```
@@ -184,7 +184,27 @@ ARG GOVERSIONINFO_VERSION
 RUN --mount=type=bind,from=gocross,source=/usr/local/go,target=/usr/local/go,rw \
     GOBIN=/out GO111MODULE=on go install "github.com/josephspurrier/goversioninfo/cmd/goversioninfo@${GOVERSIONINFO_VERSION}"
 
-FROM build-base-static AS builder-static
+FROM build-base-static AS builder-static-nosdk
+ARG DEBIAN_FRONTEND
+RUN apt-get install -y --no-install-recommends dpkg-dev clang lld llvm make pkg-config
+ARG PKG_NAME
+ARG DOCKER_CLI_REF
+ARG NIGHTLY_BUILD
+WORKDIR /build
+ARG TARGETPLATFORM
+RUN xx-apt-get install -y gcc libc6-dev
```

Not sure if it makes a difference for caching, but some questions;

- move the PKG_NAME and some of the other ones later?
- do the `apt-get` and `xx-apt-get` steps have to be separate? (would it make sense to combine them otherwise?)
- does `xx-apt...` use `TARGETPLATFORM`? If not, that one could be moved until after that step as well


```suggestion
FROM build-base-static AS builder-static-nosdk
WORKDIR /build
ARG DEBIAN_FRONTEND
RUN xx-apt-get install -y --no-install-recommends \
    clang \
    dpkg-dev \
    gcc \
    libc6-dev \
    lld \
    llvm \
    make \
    pkg-config

ARG DOCKER_CLI_REF
ARG NIGHTLY_BUILD
ARG PKG_NAME
ARG TARGETPLATFORM
```

---

> * move the PKG_NAME and some of the other ones later?

Yes good point, could be moved after `xx-apt`

> * do the `apt-get` and `xx-apt-get` steps have to be separate? (would it make sense to combine them otherwise?)

Yes they have because `xx-apt-get` is using `TARGET_PLATFORM`.

> * does `xx-apt...` use `TARGETPLATFORM`? If not, that one could be moved until after that step as well

Yes every `xx-*` command needs `TARGETPLATFORM`.